trade/assets/js/index.js

```diff
@@ -0,0 +1,156 @@
+// index.js
+
+import { fetchCurrencies, fetchTimeSeries } from './service.js';
```

Comment:
dont use index.js as module direct add it in html and remove all export that you but in all files

Reply:
done